### PR TITLE
feat: added vs code extension blog

### PR DESCRIPTION
**Pull request by Aayushd18**

### CODE REVIEW

File: content/blog/vs_code_extension_blog.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+ ---
+  
+[**FOSS Overflow**](https://fossoverflow.dev/) is a month-long event celebrating open source culture and is organized by Google Developer Student Clubs and OpenLake at **IIT Bhilai**. The program is designed to prepare students for global open source internships like GSoC, LFX, and Outreachy. Foss Overflow gave me a great opportunity to learn something new while working on a project. 
+
+<div align="center">
+
+  
+
+![Alt text](https://fossoverflow.dev/nextimg/%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png/828/75?url=%2F_next%2Fstatic%2Fmedia%2Flogo.4bd48c29.png&w=828&q=75)
```
Comment (supercoww) [inline content/blog/vs_code_extension_blog.md:25]: This image URL may break. Refer https://github.com/OpenLake/FOSS-Overflow/pull/2/files for including images

File: content/blog/vs_code_extension_blog.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+During this project I also got to learn Figma for UI design. My first design for the front-end of Activity Tracker is as follows:- 
+
+![](https://lh6.googleusercontent.com/OhVUSmrawchiNkEnZGF2ZQwaIRuWbuc4iA3oZNYGisx9LvGbQfqvVshCorGdlkr4WiywFVl9WcX8dRJYRmnXmKbnbuRBMTEJrJOsol-B5LcdDEvA0oGXjfwqbS-YTcbuC1PXT85_)
+<br/>
+
+The event gave me an opportunity to meet new people and together fix errors in code, designing UI etc. Overall a good experience sharing knowledge with each other.
+
+  
+Special thanks to my mentors [Ambar Mutha](mailto:ambarm@iitbhilai.ac.in), [Kumar Shivendu](mailto:shivendu@iitbhilai.ac.in) and Shikhar Vashistha who were there to help me with my doubts and clearing the errors in code making it more clean and efficient.
```
Comment (supercoww) [inline content/blog/vs_code_extension_blog.md:50]: Please don't make emails public 😢. You can use my GitHub.

File: content/blog/vs_code_extension_blog.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The extension not only stores the time spent but also gets the data about the time spent in writing codes in different languages. So that the user can see which language they use the most and in which they are more efficient.
+
+This Extension is a subset of the activity tracker application which tracks time spent in various applications not only vs code. It is a full stack application which allowed me to learn and understand the various aspects of developing a full stack application. Now I am more confident in developing a scalable full stack application while maintaining a clean and organized code.
+
+During this project I also got to learn Figma for UI design. My first design for the front-end of Activity Tracker is as follows:- 
+
+![](https://lh6.googleusercontent.com/OhVUSmrawchiNkEnZGF2ZQwaIRuWbuc4iA3oZNYGisx9LvGbQfqvVshCorGdlkr4WiywFVl9WcX8dRJYRmnXmKbnbuRBMTEJrJOsol-B5LcdDEvA0oGXjfwqbS-YTcbuC1PXT85_)
```
Comment (supercoww) [inline content/blog/vs_code_extension_blog.md:44]: Copy the image in repo

**Review by supercoww — CHANGES_REQUESTED** [review]
Looks good overall. Just check for grammar with Docs or Grammarly. Also include a screenshot of the implemented UI if you want to.

File: content/blog/vs_code_extension_blog.md (hunk 1 above)
Comment (Aayushd18) [reply content/blog/vs_code_extension_blog.md]: added the pic in the repo.

File: content/blog/vs_code_extension_blog.md (hunk 2 above)
Comment (Aayushd18) [reply content/blog/vs_code_extension_blog.md]: Sorry about that, added the github profile instead.

File: content/blog/vs_code_extension_blog.md (hunk 3 above)
Comment (Aayushd18) [reply content/blog/vs_code_extension_blog.md]: deno.

**Pull request merged**